test/widgets/gl_tester.cpp

```diff
@@ -8,19 +8,19 @@
 
 namespace {
 constexpr int kAnimationDuration = 5000;
-}
+} // anonymous namespace
 
 namespace QMapLibre {
 
 GLTester::GLTester(const QMapLibre::Settings &settings)
     : GLWidget(settings) {}
 
 void GLTester::initializeAnimation() {
-    m_bearingAnimation = new QPropertyAnimation(map(), "bearing");
-    m_zoomAnimation = new QPropertyAnimation(map(), "zoom");
+    m_bearingAnimation = std::make_unique<QPropertyAnimation>(map(), "bearing");
+    m_zoomAnimation = std::make_unique<QPropertyAnimation>(map(), "zoom");
 
-    connect(m_zoomAnimation, &QPropertyAnimation::finished, this, &GLTester::animationFinished);
-    connect(m_zoomAnimation, &QPropertyAnimation::valueChanged, this, &GLTester::animationValueChanged);
+    connect(m_zoomAnimation.get(), &QPropertyAnimation::finished, this, &GLTester::animationFinished);
```

Comment:
The stuff you pass to `connect()` does not outlive `GLTester` right?

Reply:
This direction is perfectly fine as when `m_zoomAnimation` is deleted, no signal will ever be fired.
Also connections usually do not need to be manually removed on destruction.

The other direction may be potentially problematic (i.e. connecting `this` to some other pointer), but in practice, as you mention, the lifetime is ~same so no problems.